### PR TITLE
doc: Fix sample config option linking

### PR DESCRIPTION
**Pull request by umapraseeda**
Fix sample config option linking
The PR temporary pulls in the associated style changes made to the theme repo (https://github.com/nrfconnect/doc-sphinx-ncs-theme/pull/4) just to make the changes visible in the PR preview (through https://github.com/nrfconnect/sdk-nrf/pull/6253/commits/dce3f1e4b5518708b368b21a6e4fdaa5547ff9b1) and this commit will be removed before merging.
Signed-off-by: Uma Praseeda <uma.praseeda@nordicsemi.no>

### CODE REVIEW

**Review by divipillai — APPROVED** [review]
All of the config options have been updated as expected and all of them are linked correctly.
looks good.

File: doc/nrf/versions.txt.in [hunk 1]
```diff
@@ -71,6 +71,6 @@
 .. |recommonmark_ver| replace:: @RECOMMONMARK_VERSION@
 .. |sphinx_ver| replace:: @SPHINX_VERSION@
 .. |sphinx-tabs_ver| replace:: @SPHINX-TABS_VERSION@
-.. |sphinx-ncs-theme_ver| replace:: @SPHINX-NCS-THEME_VERSION@
+.. |sphinx-ncs-theme_ver| replace:: 0.6.4
```
Comment (coderbyheart) [inline doc/nrf/versions.txt.in:74]: Change intentional?

Comment (umapraseeda) [reply doc/nrf/versions.txt.in:74]: Yes..this was just to see the preview with the added style.. reverting this soon to point to the new release (the style changes have been merged).

File: scripts/requirements-doc.txt [hunk 2]
```diff
@@ -1,8 +1,7 @@
 recommonmark==0.6.0
 CommonMark>=0.9.1
 sphinxcontrib-mscgen>=0.6
-sphinx-ncs-theme>=0.6.4
-pygments>=2.7.0
```
Comment (gmarull) [inline scripts/requirements-doc.txt:5]: pygments deleted?

Comment (umapraseeda) [reply scripts/requirements-doc.txt:5]: fixed this now

**Review by gmarull — APPROVED** [review]

File: samples/nrf9160/location/README.rst [hunk 3]
```diff
@@ -40,8 +40,16 @@ Configuration options
...
 Check and set the following configuration options for the sample:
 
-.. option:: CONFIG_LOCATION_SAMPLE_GNSS_ANTENNA_ONBOARD - Onboard GNSS antenna is used (default)
-.. option:: CONFIG_LOCATION_SAMPLE_GNSS_ANTENNA_EXTERNAL - External GNSS antenna is used
+.. _CONFIG_LOCATION_SAMPLE_GNSS_ANTENNA_ONBOARD:
+
+CONFIG_LOCATION_SAMPLE_GNSS_ANTENNA_ONBOARD - Configuration for onboard GNSS antenna (default)
+   This option enables the onboard GNSS antenna.
+
+
```
Comment (trantanen) [inline samples/nrf9160/location/README.rst:48]: Should we have just one empty line here?

**Pull request merged**